content/reference/current/installation/docker/index.md

```diff
@@ -95,6 +103,16 @@ Finally, we need to prepare a folder called `frontline-keys` that will hold uplo
 
 You can leave it empty.
 
+### Prepare repositories cache folder
+
+This step is optional.
+
+Build a Gatling project for the first time using Maven, Gradle or SBT will downloading some dependencies that will be kept on disk. It is possible to make a volume of these folders to skip the downloading part when spinning a brand new Gatling Enterprise container.
```

Comment:
```suggestion
Building a Gatling project for the first time, using Maven, Gradle, or SBT, will download some dependencies that will be stored on disk. It is possible to make a volume of these folders to avoid downloading the same files again each time you spin up a new Gatling Enterprise container.
```

WDYT?